app/gui/src/dashboard/components/styled/Tabs.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file
+ *
+ * This file is a modified version of the `Tabs` components from the
+ * `react-aria-components` library. We use simplified solution that better
+ * fits our needs and doesn't cause performance issues.
+ *
+ * Also, this file re-exports the `Tabs` component from `react-aria-components`
+ */
+
+import { createHideableComponent } from '@react-aria/collections'
+import type { ForwardedRef } from 'react'
+import { useContext } from 'react'
+import { useFocusRing } from 'react-aria'
+import {
+  UNSTABLE_CollectionRendererContext as CollectionRendererContext,
+  UNSTABLE_DefaultCollectionRenderer as DefaultCollectionRenderer,
+  TabListStateContext,
+  TabsContext,
+  type TabPanelProps,
+} from 'react-aria-components'
+import invariant from 'tiny-invariant'
+import { mergeProps } from '../aria'
+
+/**
+ * A TabPanel provides the content for a tab.
+ *
+ * This component is a modified version of the `TabPanel` component from the
+ * `react-aria-components` library. We use simplified solution that better
+ * fits our needs and doesn't cause performance issues.
```

Comment:
recommend saying here what this does differently that makes it faster

Reply:
Updated the description: https://github.com/enso-org/enso/blob/7ce4eb2dfb5e9099f7a901a8fa9b7208e8c5db5a/app/gui/src/dashboard/components/aria.tsx#L53-L54